docs/en/index.md

```diff
@@ -6,109 +6,109 @@
-environment for R and Python. Use the r-studio-cpu image to get an R Studio
-environment.
+!!! info annotate "AAW Portal Homepage"
+    - [**Portal Homepage for Statistics Canada Employees**](https://www.statcan.gc.ca/data-analytics-service/aaw)
+    - [**Portal Homepage for External Users**](https://analytics-platform.statcan.gc.ca/covid19)
 
-## Run a virtual desktop
+!!! info "Kubeflow Dashboard"
+    - [**Kubeflow Dashboard**](https://kubeflow.aaw.cloud.statcan.ca/) Use this link once you have your cloud account!
 
-[![Virtual Desktop](images/VirtualDesktop.PNG)](1-Experiments/Remote-Desktop.md)
-You can run a full Ubuntu desktop, with typical applications, right inside your
-browser, using [**ML Workspaces**](1-Experiments/Remote-Desktop.md)
+Getting started with the Advanced Analytics Workspace (AAW) is easy and quick. First, you'll want to set up Kubeflow for MLOps and Jupyter Notebooks. Kubeflow makes it easy to deploy and manage end-to-end machine learning workflows, while Jupyter Notebooks provide a flexible and powerful environment for data analysis and experimentation. Once you have Kubeflow and Jupyter Notebooks set up, you can start exploring the many resources and tools available through the AAW portal. Additionally, we encourage you to join our Slack channel to connect with other data scientists and analysts, ask questions, and share your experiences with the AAW platform.
 
-# Publishing
+### Kubeflow Account
 
-## Build and publish an interactive dashboard
+!!! important "Attention External Users!"
+    Users external to Statistics Canada will require a cloud account granted access by the business sponsor.
 
-[![InteractiveDashboard](images/InteractiveDashboard.PNG)](/2-Publishing/R-Shiny/)
-Use **[R-Shiny](/2-Publishing/R-Shiny/)** to build interactive web apps straight
-from R. You can deploy your R-Shiny dashboard by submitting a pull request to
-our [R-Dashboards GitHub repository](https://github.com/StatCan/R-dashboards).
-![R Shiny Server](images/readme/shiny_ui.png)
+!!! note ""
+    <center>
+    [![Kubeflow is the Heart of the AAW!](./images/Kubeflow.PNG)](https://kubeflow.aaw.cloud.statcan.ca/)
+    <h3>**[👉 Click here to setup your Kubeflow account! 👈](https://kubeflow.aaw.cloud.statcan.ca/)**</h3>
+    </center>
 
-**[Dash](/2-Publishing/Dash/)** is a data visualization tool that lets you build
-an interactive GUI around your data analysis code.
+**[Kubeflow](1-Experiments/Kubeflow/)** is a powerful and flexible open source platform that has revolutionized the way we build, deploy, and manage machine learning workflows. With its robust set of tools and frameworks, Kubeflow is the perfect starting point for data scientists and developers who want to accelerate their projects and bring them to the next level. By using up Kubeflow, you can harness the full potential of cloud-native machine learning and unlock new opportunities for innovation and growth.
```

Comment:
No ML as it currently stands. Last sentence is a little iffy.  Maybe instead  "Using Kubeflow allows dynamic leverage of cloud compute, with users having the ability to control compute, memory, and storage resources used." Not entirely sure what the intent of this sentence is, but from my perspective, when I see lots of abstract talk I get skeptical

Reply:
Agreed.